.github/workflows/depviz.yml

```diff
@@ -0,0 +1,15 @@
+on:
+  workflow_dispatch:
+  push:
```

Comment:
« Push » isn’t very useful.

can you switch it to a crib tab instead? I.e once a day?

Reply:
:+1: